tests/use_case_test/update_rating/UpdateRatingInteractorTest.java

```diff
@@ -0,0 +1,45 @@
+package use_case_test.update_rating;
+
+import entity.Movie;
+import interface_adapters.update_rating.UpdateRatingPresenter;
+import interface_adapters.update_rating.UpdateRatingState;
+import use_case.movie_info.*;
+import use_case.remove_rating.*;
+import use_case.update_rating.*;
+import interface_adapters.update_rating.UpdateRatingViewModel;
+import interface_adapters.update_rating.UpdateRatingController;
+import org.junit.Test;
+import utility.ApiInterface;
+import utility.OMDBCaller;
+import entity.UserRating;
+import data_access.UserRatingAccessObject;
+
+import java.io.File;
+
+import static org.junit.Assert.*;
+
+public class UpdateRatingInteractorTest {
+    //Tests if a rating is properly updated. A userRating is first initialized to have a rating of 3.
+    //The interactor is called and updates the database, the new rating should be updated to 3.
+    @Test
+    public void testUpadateRating() {
+        String imdb = "tt0345950";
+        String username = "Alex";
+        int r = 5;
+        UserRatingAccessObject ratingAccessObject = new UserRatingAccessObject("./userRatings.csv");
```

Comment:
Same comment on line 62 of RemoveRatingInteractorTest, consider using a different file path.